frontend/src/components/Director.vue

```diff
@@ -87,7 +87,18 @@ export default {
       console.log({ transcript, isTriggered })
 
       if (isTriggered) this.skipQuestion()
-    }
+    },
+    changeDirectorsLanguage(language) {
+      const { speechRecognitionInstance } = this
+
+      if (language === 'Turkish') {
+        speechRecognitionInstance.language = 'tr-TR'
+        this.user.director.language = 'tr-TR'
```

Comment:
we can combine these two lines—`speechRecognitionInstance.language = this.user.director.language = 'tr-TR'`